assets/src/data/content/activities/DeliveryState.ts

```diff
@@ -0,0 +1,162 @@
+import { AnyAction, createSlice, PayloadAction, ThunkAction } from '@reduxjs/toolkit';
```

Comment:
This file will be split out into common state slices that are used across activities and CATA-specific slices in a following PR.